avocado/utils/pmem.py

```diff
@@ -302,16 +300,17 @@ def enable_namespace(self, namespace="all", region="", bus="", verbose=False):
         """
         args = namespace
         if region:
-            args = f"{args} -r {region}"
+            args += f" -r {region}"
         if bus:
-            args = f"{args} -b {bus}"
+            args += f" -b {bus}"
```

Comment:
Actually, you're adding on operation here.  Initially you have one "f string" operation (with two variables).  Now you have both, one "f string" and a string concatenation.  

I'm sorry but this is not an optimization.

Even worse, there's a consensus around the fact that usually [string concatenations are slower than "f string"](https://stackoverflow.com/questions/38722105/format-strings-vs-concatenation) operations.

Reply:
This is not a very good optimization, thank you for your comment